definitions/npm/autobind-decorator_v2.x.x/flow_v0.25.x-/autobind-decorator_v2.x.x.js

```diff
@@ -0,0 +1,20 @@
+declare module 'autobind-decorator' {
+    declare export interface TypedPropertyDescriptor<T> {
+      enumerable?: boolean;
+      configurable?: boolean;
+      writable?: boolean;
+      value?: T;
+      get?: () => T;
+      set?: (value: T) => void;
+    }
+
+    declare export type ClassDecorator = <TFunction: Function>(target: TFunction) => TFunction;
+    declare export type MethodDecorator = <T>(target: Object, propertyKey: string | Symbol, descriptor: TypedPropertyDescriptor<T>) => TypedPropertyDescriptor<T>;
```

Comment:
Same for `Object`. `{}` should do it.

Reply:
Fixed cff1b4bda547fd8eb35c415d5ea4af4a8326bbc0